README.md

```diff
@@ -42,94 +58,43 @@ Below is a sample playbook:
       force_reactivation: false
 ```
 
-Please refer to the [Variables](#variables) section for required variables.
-
-
-## Variables<a id="variables"></a>
-
-
-#### Task : deploy.yml
-###### action: deploy
-
-The <code>deploy</code> task includes the <code>install</code> and <code>activate</code> playbooks internally.
-
-Key | Type | Description | Default
+<a id="actions"></a>
+## Actions
+Definition for possible actions to be performed in this role, for required variables please refer to [Variables](#variables).
+
+Action | Description | Variables
+------------|-------------|-------------------
+deploy | The <code>deploy</code> task includes the <code>install</code> and <code>activate</code> playbooks internally. | dsm_agent_download_hostname <br /> dsm_agent_download_port <br /> dsm_agent_activation_hostname	 <br/> dsm_agent_activation_port <br/> policy_id (Optional) <br/> group_id (Optional) <br/> force_reactivation (Optional) <br/><br/> **For Multi-Tenancy:** <br/> tenant_id <br /> token/tenant_password
+install | The install task downloads and installs the Deep Security Agent. The installation is skipped if an agent of the same version is already installed. If a newer version of Deep Security Installer is already installed, then the version is upgraded. | dsm_agent_download_hostname <br/> dsm_agent_download_port
+activate | The activate task activates the Deep Security Agent by registering it in Trend Micro Deep Security Manager. By default, activation is skipped if the agent is already activated, unless the force_reactivation attribute is set to true. | dsm_agent_activation_hostname <br/> dsm_agent_activation_port <br/> policy_id (Optional) <br/> group_id (Optional) <br/> force_reactivation (Optional) <br/><br/> **For Multi-Tenancy:** <br/> tenant_id <br/> token/tenant_password
+set-policy-by-name | Change agent's policy via policy name | policy_id
+set-policy-by-id | Change agent's policy via policy id | policy_name
```

Comment:
Change agent's -> Change the agent's (change both instances)
via -> via the (change both instances)